dev-libs/libphonenumber/libphonenumber-8.12.26.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR=emake
+
+inherit cmake java-pkg-2
+
+DESCRIPTION="library for parsing, formatting, and validating international phone numbers"
+HOMEPAGE="https://github.com/google/libphonenumber"
+SRC_URI="https://github.com/google/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64 ~arm64"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+
+RDEPEND="
+	dev-libs/boost
+	dev-libs/icu
+	dev-libs/protobuf
+"
+
+BDEPEND="
+	dev-cpp/gtest
+	virtual/jdk
```

Comment:
`DEPEND=">=virtual/jdk-1.8:*"`
If Java files are installed, then also: `RDEPEND=">=virtual/jre-1.8:*"`

Even in `EAPI="7"` or `EAPI="8"`, java-config still parses `DEPEND` instead of `BDEPEND`:
https://gitweb.gentoo.org/proj/java-config.git/tree/src/java_config_2/Package.py?id=82a5a6343e7b6172af6aaa1b3ea0a61a43afd206#n68

Reply:
Yeah, java files have to be installed but actually I'm not able to handle them, yet

Reply:
Java is not required at the end